sunpy/map/mapbase.py

```diff
@@ -1559,66 +1561,66 @@ def draw_limb(self, axes=None, **kwargs):
 
         return [circ]
 
+    @deprecate_positional_args
     @u.quantity_input
-    def draw_rectangle(self, bottom_left, width: u.deg, height: u.deg, axes=None, **kwargs):
+    def draw_rectangle(self, bottom_left, *, width: u.deg=None, height: u.deg=None,
```

Comment:
We need to open an issue to re-order and remove the deprecation on these methods after the 2.0 release.

Reply:
[Done here!](https://github.com/sunpy/sunpy/issues/4025)